File: java/models/commande.java
package models;

import java.util.Date;

public class commande  {
    private  int id;
    private float prix_coomande;
    private String name_commande;
    private Date date_commande;
    private String payment;
    private int id_oeuvre;

    public int getId_oeuvre() {
        return id_oeuvre;
    }

    public void setId_oeuvre(int id_oeuvre) {
        this.id_oeuvre = id_oeuvre;
    }
    public int getID() {
        return id;
    }

    public void setID(int id) {
        this.id=id;
    }



    public float getprix_commande() {
        return prix_coomande;
    }

    public void setprix_commande(float type) {
        this.prix_coomande = type;
    }

    public String getname_commande() {
        return name_commande;
    }

    public void setname_commande(String name_commande) {
        this.name_commande = name_commande;
    }

    public Date getdate_commande() {
        return date_commande;
    }
    public void setdate_commande(Date date) {
        this.date_commande = date;
    }
    public String getpayment() { return payment; }

    public void setpayment(String payment) {
        this.payment = payment;
    }


    // Constructors, getters, and setters

    public commande() {

    }
    public commande(int id, float prix_coomande, String name_commande, Date date_commande ,String payment, int id_oeuvre)  {
        this.id=id;
        this.prix_coomande = prix_coomande;
        this.name_commande = name_commande;
       this.date_commande=date_commande;
        this.payment = payment;
        this.id_oeuvre=id_oeuvre;

    }


    public commande(float prix_coomande, String name_commande, String payment,int id_oeuvre)  {
        this.prix_coomande = prix_coomande;
        this.name_commande = name_commande;

        this.payment = payment;
        this.id_oeuvre=id_oeuvre;

    }

    // Getters and Setters




}

File: src/Entity/Commande.php
<?php

namespace App\Entity;

use Doctrine\DBAL\Types\Types;
use Doctrine\ORM\Mapping as ORM;

/**
 * Commande
 *
 * @ORM\Table(name="commande", indexes={@ORM\Index(name="id_user", columns={"id_user"}), @ORM\Index(name="fk_produit_commande", columns={"id_oeuvre"})})
 * @ORM\Entity
 */
class Commande
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_commande", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idCommande;

    /**
     * @var string
     *
     * @ORM\Column(name="name_commande", type="string", length=255, nullable=false)
     */
    private $nameCommande;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_commande", type="date", nullable=false)
     */
    private $dateCommande;

    /**
     * @var float
     *
     * @ORM\Column(name="prix_commande", type="float", precision=10, scale=0, nullable=false)
     */
    private $prixCommande;

    /**
     * @var string
     *
     * @ORM\Column(name="paiement", type="string", length=20, nullable=false)
     */
    private $paiement;

    /**
     * @var int|null
     *
     * @ORM\Column(name="Quantity", type="integer", nullable=true)
     */
    private $quantity;

    /**
     * @var \Oeuvre
     *
     * @ORM\ManyToOne(targetEntity="Oeuvre")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_oeuvre", referencedColumnName="id_oeuvre")
     * })
     */
    private $idOeuvre;

    /**
     * @var \User
     *
     * @ORM\ManyToOne(targetEntity="User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_user", referencedColumnName="user_id")
     * })
     */
    private $idUser;

    public function getIdCommande(): ?int
    {
        return $this->idCommande;
    }

    public function getNameCommande(): ?string
    {
        return $this->nameCommande;
    }

    public function setNameCommande(string $nameCommande): static
    {
        $this->nameCommande = $nameCommande;

        return $this;
    }

    public function getDateCommande(): ?\DateTimeInterface
    {
        return $this->dateCommande;
    }

    public function setDateCommande(\DateTimeInterface $dateCommande): static
    {
        $this->dateCommande = $dateCommande;

        return $this;
    }

    public function getPrixCommande(): ?float
    {
        return $this->prixCommande;
    }

    public function setPrixCommande(float $prixCommande): static
    {
        $this->prixCommande = $prixCommande;

        return $this;
    }

    public function getPaiement(): ?string
    {
        return $this->paiement;
    }

    public function setPaiement(string $paiement): static
    {
        $this->paiement = $paiement;

        return $this;
    }

    public function getQuantity(): ?int
    {
        return $this->quantity;
    }

    public function setQuantity(?int $quantity): static
    {
        $this->quantity = $quantity;

        return $this;
    }

    public function getIdOeuvre(): ?Oeuvre
    {
        return $this->idOeuvre;
    }

    public function setIdOeuvre(?Oeuvre $idOeuvre): static
    {
        $this->idOeuvre = $idOeuvre;

        return $this;
    }

    public function getIdUser(): ?User
    {
        return $this->idUser;
    }

    public function setIdUser(?User $idUser): static
    {
        $this->idUser = $idUser;

        return $this;
    }


}
